src/types/runtime/gravity/Video.ts

```diff
@@ -0,0 +1,31 @@
+import { GraphQLInt, GraphQLObjectType, GraphQLString } from "graphql"
+import { Number, Record, String } from "runtypes"
+import { ResolverContext } from "../../graphql"
+
+export const VideoType = new GraphQLObjectType<any, ResolverContext>({
+  name: "Video",
+  description: "An object containing video metadata",
+  fields: {
+    src: {
+      description: "The url of the video",
+      type: GraphQLString,
+      resolve: ({ src }) => src,
+    },
+    height: {
+      description: "The height of the video",
+      type: GraphQLInt,
+      resolve: ({ height }) => height,
+    },
+    width: {
+      description: "The width of the video",
+      type: GraphQLInt,
+      resolve: ({ width }) => width,
```

Comment:
I believe these resolvers are redundant because by default graphql.js will use the field name for object access.

Reply:
Yup, the only reason we ever need to resolve fields like this is if we're accessing `snake_case` props from Gravity that we' by convention convert to `camelCase`. 

So like: 
```tsx
someField: {
  type: ... 
  resolve: ({ some_field }) => some_field
}
```